command-expansion-server/src/app.ts

```diff
@@ -135,7 +135,13 @@ app.post('/put-dictionary', async (req, res, next) => {
     throw new Error(`POST /dictionary: No command dictionary was updated in the database`);
   }
   const id = rows[0].id;
-  res.status(200).json({ id });
+  const command_types_typescript_path = rows[0].command_types_typescript_path;
```

Comment:
If you don't want to keep accessing `rows[0]` here you make this a one-liner using destructuring assignment:

```ts
const [{ command_types_typescript_path, created_at, id, mission, updated_at, version }] = rows;
```

Reply:
Similarly if the first `row` has all the properties we need you could do:

```ts
const [row] = rows;
res.status(200).json(row);
```

Reply:
Thanks for the suggestion. I will make that change.